.drone.yml

```diff
@@ -0,0 +1,153 @@
+workspace:
+  base: /var/www/owncloud
+  path: apps/diagnostics
+
+branches: [ master ]
+
+clone:
+  git:
+    image: plugins/git:next
+
+pipeline:
+  install-server:
+    image: owncloudci/core
+    pull: true
+    version: daily-stable10-qa
```

Comment:
either use `${OC_VERSION}` or remove the duplicate definitions in the matrix configuration

Reply:
@patrickjahns I am not sure where should I define it. Can you give me some example so I get an idea?

Reply:
@patrickjahns ping